server/utils/db/connection_pool.py

```
@@ -45,9 +45,9 @@ def __init__(self):
             if pg_sslrootcert:
                 self.db_params['sslrootcert'] = pg_sslrootcert
 
-        # Connection pool configuration
+        # Keep max low: 5 services × 10 = 50, well under PostgreSQL's default max_connections (100).
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)